.github/workflows/release.yml

```diff
@@ -23,6 +23,11 @@ jobs:
         with:
           go-version-file: go.mod
       
+      - name: Install ImageMagick
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y imagemagick
+      
```

Comment:
Could we instead just somehow split the icon generation and the metadata generation so the icon generation go generate commands don't need to run in CI?